dora/core/client/fs/src/test/java/alluxio/client/file/dora/ConsistentHashPolicyTest.java

```diff
@@ -69,21 +73,25 @@ public void getMultipleWorkers() throws Exception {
     // Prepare a worker list
     List<BlockWorkerInfo> workers = new ArrayList<>();
     WorkerNetAddress workerAddr1 = new WorkerNetAddress()
-            .setHost("master1").setRpcPort(29998).setDataPort(29999).setWebPort(30000);
-    workers.add(new BlockWorkerInfo(workerAddr1, 1024, 0));
+        .setHost("master1").setRpcPort(29998).setDataPort(29999).setWebPort(30000);
+    final WorkerIdentity identity = WorkerIdentityTestUtils.ofLegacyId(1);
+    workers.add(new BlockWorkerInfo(
+        identity, workerAddr1, 1024, 0));
     WorkerNetAddress workerAddr2 = new WorkerNetAddress()
-            .setHost("master2").setRpcPort(29998).setDataPort(29999).setWebPort(30000);
-    workers.add(new BlockWorkerInfo(workerAddr2, 1024, 0));
+        .setHost("master2").setRpcPort(29998).setDataPort(29999).setWebPort(30000);
+    workers.add(new BlockWorkerInfo(
+        WorkerIdentityTestUtils.ofLegacyId(2), workerAddr2, 1024, 0));
 
     List<BlockWorkerInfo> assignedWorkers = policy.getPreferredWorkers(workers, "hdfs://a/b/c", 2);
     assertEquals(2, assignedWorkers.size());
     assertTrue(assignedWorkers.stream().allMatch(w -> contains(workers, w)));
     // The order of the workers should be consistent
-    assertEquals(assignedWorkers.get(0).getNetAddress().getHost(), workerAddr2.getHost());
-    assertEquals(assignedWorkers.get(1).getNetAddress().getHost(), workerAddr1.getHost());
+    assertEquals(assignedWorkers.get(0).getNetAddress().getHost(), workerAddr1.getHost());
+    assertEquals(assignedWorkers.get(1).getNetAddress().getHost(), workerAddr2.getHost());
```

Comment:
This was a bug?

Reply:
Not really. This PR changes the way worker identity is hashed, so the relative positions of the workers in the hash ring are affected. So this test should be updated accordingly.